scalableminds/webknossos
# Edit task consistency

### Steps to test:
- try to edit a task: you should only be able to edit "Task Instances"
- try to create a task: you should be able to edit all fields

### Issues:
- fixes #2198

------
